### PR TITLE
AIW-1971 - Docs for Logo recognition flow

### PR DESCRIPTION
**Pull request by aMacDawg**

### CODE REVIEW

File: src/docs/developer/flow/README.md [hunk 1]
```diff
@@ -44,3 +44,25 @@ Your debugging experience in Automate Studio can be thought of as two steps:
...
+This flow example illustrates how you can publish flows in Automate Studio as cognitive engines that introduce new cognitive models onto aiWARE.
+
+*Note: There are multiple ways to creative cognitive engines, including running the cognitive model and wrapping in the engine or (as seen in this example) making calls to a third party API to send each chunk message (eg, a slice of a file) to that service for processing*
+
+**Steps**
+
+1. Like any flow in Automate Studio, we can read the logic from left to right like reading a book, and let’s examine this flow by slicing it up into thirds
+2. In the first third of this flow, we see two http in nodes as well as an http response node. These nodes are part of the webhook pattern for how the flow engine communicates with aiWARE at runtime. 
+3. This is what the engine toolkit expects, and all flow engines need to conform to this design to successfully run as engines on aiWARE
+4. The function node performs a filtering function, checking the msg.payload properties of the chunk msg injected into the flow engine at runtime and evaluating if the message is indeed a “media_chunk” and the “mimeType” is “image/jpeg”
```
[the remark this answers is not in the document]
Comment (aMacDawg) [reply src/docs/developer/flow/README.md]: > Will this eventually get screen shots? Seems like it needs some.
> 
> Add images to the /flow folder and use markdown like this to insert images into the page:
> 
> `Alt text goes here`

Thanks @kasthomas2 ! Yes, the plan is to include screenshots of the flow for this and other flow tutorials

File: src/docs/developer/flow/README.md [hunk 2]
```diff
@@ -44,3 +44,25 @@ Your debugging experience in Automate Studio can be thought of as two steps:
...
+#### Logo Recognition
+
+This flow example illustrates how you can publish flows in Automate Studio as cognitive engines that introduce new cognitive models onto aiWARE.
+
+*Note: There are multiple ways to creative cognitive engines, including running the cognitive model and wrapping in the engine or (as seen in this example) making calls to a third party API to send each chunk message (eg, a slice of a file) to that service for processing*
+
+**Steps**
+
+1. Like any flow in Automate Studio, we can read the logic from left to right like reading a book, and let’s examine this flow by slicing it up into thirds
+2. In the first third of this flow, we see two http in nodes as well as an http response node. These nodes are part of the webhook pattern for how the flow engine communicates with aiWARE at runtime.
```
[the remark this answers is not in the document]
Comment (aMacDawg) [reply src/docs/developer/flow/README.md]: I think backticks might make sense, my issue is that the code styling looks fancy but gets noisy quick. Fyi, in node-red parlance, the node names are usually all lowercase, so I am respecting that styling, but an _italicization_ might make sense

File: src/docs/developer/flow/README.md [hunk 3]
```diff
@@ -44,3 +44,25 @@ Your debugging experience in Automate Studio can be thought of as two steps:
...
+
+### Flow Engine examples
+
+Let's get started with some walkthroughs of introductory and real use-cases of flow engines currently in use by aiWARE customers
+
+#### Logo Recognition
+
+This flow example illustrates how you can publish flows in Automate Studio as cognitive engines that introduce new cognitive models onto aiWARE.
+
+*Note: There are multiple ways to creative cognitive engines, including running the cognitive model and wrapping in the engine or (as seen in this example) making calls to a third party API to send each chunk message (eg, a slice of a file) to that service for processing*
```
[the remark this answers is not in the document]
Comment (aMacDawg) [reply src/docs/developer/flow/README.md]: Wunderbar, thanks!

File: src/docs/developer/flow/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kasthomas2) [reply src/docs/developer/flow/README.md]: I'm not fussy about backticks vs. italics, but I would say there does need to be SOME kind of visual cue to this. All lowercase is fine as long as there is _some_ kind of visual signal to the user that the name is a special label.

File: src/docs/developer/flow/README.md [hunk 4]
```diff
@@ -44,3 +44,25 @@ Your debugging experience in Automate Studio can be thought of as two steps:
...
+5. If there are chunk msgs injected into the flow engine at runtime that do not meet the criteria, the if statement formats an ignore msg that is then sent back as the http response at runtime.
+6. The middle third of the flow is the most straight forward. Starting with the “set payload and headers” function node, we prepare the msg details for making an API http request to the Logo Recognition service, in this case, Google’s Logo Recognition model
+7. Please note: you will need your own Google API key if you are trying to run and publish your own version of this flow engine
+8. Each image chunk is sent to the Google Logo API and returns a response that the http request node captures and sends to the “transform VTN” function node, which parses the response, and assuming the response was successful, outputs to the chunk msg to the “output to Edge” http response node.
+9. Behind the scenes, aiWARE Edge will stitch the results of the Logo Recognition process back together and write to the Temporal Data Object media file that the user invoked the engine to run on.
```
[the remark this answers is not in the document]
Comment (aMacDawg) [reply src/docs/developer/flow/README.md]: good call, thanks!

**Pull request merged**